define-and-ideate/README.md

```diff
@@ -0,0 +1,40 @@
+
+The customer requests that a software development team provide a solution that ingests and consolidates data from multiple and varied existing data sources (email, spreadsheets) and then provides a daily report of new issues.
+
+The true solution however - the most meaningful result - might be to provide a free mobile app that allows citizens to easily report a road issue (pothole, debris, etc.) which loads a centralized queue for dispatchers.  The app allows the road crew to report real-time status of repairs.
+
+In this example, the customer's identification of the problem is influenced by the immediate pain point of having too much incoming data in too many formats.  The designer, however, delves to the root problem and envisions a more innovative solution.  One that not only addresses the unmanaged data, but provides real-time feedback to the citizen, dispatcher, and road crew so that nothing falls through the cracks and everyone is accountable for the condition of the roads.
+<hr>
+</details>
+
+Looking back at our case study, our customer likely presented a problem statement similar to the following:
+
+ &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_We need a system to ingest and consolidate road maintenance issues from multiple sources and in many formats._
+
+But is that the core problem, or just a symptom?  Wouldn't a more **_meaningful_** and **_actionable_**, **_human-centric_** problem statement be something like this:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_The citizens in our town need to be able easily report road issues and follow the progress of repairs._
+
+<hr>
+
+After the problem has been defined, designers can begin to generate ideas for a solution.  This is phase 3 - **Ideate**, and this is the place where innovation happens!  The team generates the widest possible range of ideas - creativity is key. The solutions will be refined in a later phase.
+
+
+
+<img src="https://user-images.githubusercontent.com/57373296/76125872-bcdfba00-5fcb-11ea-8c74-05867cfc8b1e.png" width="250" height="250">
+</p>
+To transition to the Ideate phase from the Define phase, teams often ask "how might we..." - so in this case our team might ask "how might we provide our citizens with an easy way to report road hazards and see the progress of the repairs?"
+
+
+Here are some additional resources for Ideation techniques (some of them may surprise you, like [worst possible idea](https://youtu.be/XOA6s6YCKnQ)!):
+- [Ideation](https://www.interaction-design.org/literature/topics/ideation/)
+- [Key Ideation Techniques](https://careerfoundry.com/en/blog/ux-design/what-is-ideation-in-design-thinking)
```

Comment:
Gather , collect , organize data
analyze data to identify / define the problem / pain point of users.
This will in turn help to create the solution.